server/controllers/userController.js

```diff
@@ -84,15 +105,9 @@ let UserController = function (userRepository) {
         if (!user) {
           res.status(400).json({message: "No user with that id"});
         } else {
-          const feedbackMessage = (`
-New user feedback received from ${user.firstName} ${user.lastName} (${user.email}).
-Message sent at ${moment().format('MMMM Do YYYY, h:mm:ss a')}
-
-\"${req.body.feedback}\"
-
-Love from Mobilise.
-          `.trim());
-
+          const feedbackMessage = (`\nNew user feedback received from ${user.firstName} ${user.lastName} (${user.email})
```

Comment:
why ):

Reply:
Gimme a sec

Reply:
Basically was dealing with some IDE warning, but tbh I can still make it nicer